lessons/pydata/eda-univariate-timeseries/index.ipynb

```diff
@@ -0,0 +1,4745 @@
+{
```

Comment:
<blockquote>V našem případě denních dat to sice není zcela zásadní.</blockquote><p> </p><p>Tady by se dalo namítnout, že datum fakt časovou zónu nemá a že 1. ledna a 1. ledna 0:00:00 není jen jinak přesně zapsaná hodnota (v pandas se to tak tváří :-( ), ale něco fundamentálně odlišného.  A že to nejen není zásadní, ale je to vlastně špatně.   (just thinkin').</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Určitě jste si všimli, že přibyl u údaj o čase, a samozřejmě o časové zóně a tím i o posunu od <a href="https://cs.wikipedia.org/wiki/Koordinovan%C3%BD_sv%C4%9Btov%C3%BD_%C4%8Das" target="_blank">UTC</a>. <code>+01:00</code> znamená +1 hodina od UTC. U datumů je totiž koncept časové zóny ne úplně přirozený a proto Pandas přidal i čas (<code>00:00:00</code>). Jelikož je ale často význam denních dat nějaký souhrn za konkrétních 24 hodin (a někdy za 23 hodin a někdy za 25 hodin díky střídaní letního a zimního času), je lepší pracovat i konkrétním časem (začátkem dne). A to je i náš případ.</p>